clients/rospy/src/rospy/core.py

```diff
@@ -109,7 +110,7 @@ def parse_rosrpc_uri(uri):
     @raise ParameterInvalid: if uri is not a valid ROSRPC URI
     """
     if uri.startswith(ROSRPC):
-        dest_addr = uri[len(ROSRPC):]            
+        dest_addr = uri[len(ROSRPC):]

```

Comment:
Please avoid any unrelated white space changes in your PR (even if the current file might be "ugly"). It makes backporting changes across branches much more difficult.